homeassistant/components/solax/sensor.py

```diff
@@ -15,24 +17,26 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({vol.Required(CONF_IP_ADDRESS): cv.string})
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_IP_ADDRESS): cv.string,
+        vol.Optional(CONF_PORT, default=80): cv.port,
```

Comment:
Use:

```
DEFAULT_PORT = 80
...
    vol.Optional(..., default=DEFAULT_PORT): ...
```

Reply:
I do not agree with this. On the same line it includes the words port, conf, optional, and default. It should be clear to anyone reading the code in the future that the value 80 represents the default port, provided by the config.

Reply:
Oh bugger, it's `DEFAULT_PORT` in many of the other integrations. I'll pull my head in.